### PR TITLE
Update newclient.html

### PR DESCRIPTION
**Pull request by trae1357**
update to new client like colors to match other pages and functions

### CODE REVIEW

File: tortilla/client/public/newclient.html [hunk 1]
```diff
@@ -10,19 +10,19 @@
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-wEmeIV1mKuiNpC+IOBjI7aAzPcEZeedi5yW5f2yOq55WWLwNGmvvx4Um1vskeMj0" crossorigin="anonymous">
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
   </head>
-  <div class="jumbotron bg-primary text-white">
-    <h1 class="display-6 bg-primary text-white">Welcome to your one stop shop for anything tortilla.</h1>
+  <div class="jumbotron bg-warning text-black">
```
Comment (Aubsma) [inline tortilla/client/public/newclient.html:13]: ok

File: tortilla/client/public/newclient.html [hunk 2]
```diff
@@ -10,19 +10,19 @@
...
+    <h1 class="display-6 bg-warning text-black">Welcome to your one stop shop for anything tortilla.</h1>
```
Comment (Aubsma) [inline tortilla/client/public/newclient.html:14]: ok

File: tortilla/client/public/newclient.html [hunk 3]
```diff
@@ -10,19 +10,19 @@
...
     <p>Already a user? Sign in here!.</p>
     <p class="lead">
       <a class="btn bg-success text-white btn-lg" href="#" role="button">Sign in</a>
     </p>
   </div>
-  <form>
-    <div class="form-group">
-      <label for="emailAddress">Email address</label>
-      <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp" placeholder="Enter email">
+  <form class= "bg-">
```
Comment (Aubsma) [inline tortilla/client/public/newclient.html:22]: ok

File: tortilla/client/public/newclient.html [hunk 4]
```diff
@@ -10,19 +10,19 @@
...
+    <div class="form-group bg-">
```
Comment (Aubsma) [inline tortilla/client/public/newclient.html:23]: ok

File: tortilla/client/public/newclient.html [hunk 5]
```diff
@@ -10,19 +10,19 @@
...
+      <label for="emailAddress bg-">Email address</label>
```
Comment (Aubsma) [inline tortilla/client/public/newclient.html:24]: ok

File: tortilla/client/public/newclient.html [hunk 6]
```diff
@@ -10,19 +10,19 @@
...
+      <input type="email bg-danger" class="form-control bg-muted" id="exampleInputEmail1" aria-describedby="emailHelp" placeholder="Enter email">
```
Comment (Aubsma) [inline tortilla/client/public/newclient.html:25]: ok

**Review by Aubsma — APPROVED** [review]
ok

**Comment by Aubsma** [comment]
okay

**Comment by trae1357** [comment]
Old

**Pull request closed without merging**